testsuite/integration/manualmode/src/test/java/org/wildfly/test/manual/elytron/seccontext/WildFlyElytronClientDefaultSSLContextProviderTestCase.java

```diff
@@ -63,7 +63,6 @@
  * WildFlyElytronClientDefaultSSLContextProvider is configured on client side to provide default SSL context that RESTEsy client utilizes to connect to the server.
  */
 @RunWith(Arquillian.class)
-@Ignore("WFLY-16633")
```

Comment:
@Skyllarr This fails checkstyle; I assume because you need to remove L55 'import org.junit.Ignore;'

Reply:
@bstansberry Fixed. Thank you